Doc/library/pickle.rst

```diff
@@ -235,12 +236,12 @@ process more convenient:
 
 .. function:: loads(bytes_object, \*, fix_imports=True, encoding="ASCII", errors="strict", buffers=None)
 
-   Read a pickled object hierarchy from a :class:`bytes` object and return the
-   reconstituted object hierarchy specified therein.
+   Return the reconstituted object hierarchy of the pickled representation
+   *bytes_object* of an object.
```

Comment:
Any particular reason that you changed :class:`bytes` object to *bytes_object*?

Reply:
Yes, it is because contrary to all other parameters, pickle.loads's *bytes_object* parameter was not referenced in the function description (and the type :class:`bytes` is already given by the phrase "pickled representation").